examples/batch/main.tf

```diff
@@ -83,3 +83,26 @@ EOF
     version   = "latest"
   }
 }
+
+data "azurerm_image" "image" {
```

Comment:
Could we split this out into another folder with a complete working example that doesn't depend on an existing image?

Reply:
Done.